system/jlib/jutil.hpp

```diff
@@ -556,5 +559,11 @@ class jlib_decl QuantileFilterIterator
     bool isQuantile;
 };
 
+template <typename Iterator, typename Value>
+inline bool stdIterContains(Iterator&& iter, Value &&v)
```

Comment:
picky: I'm not sure why iter is in the name.   Iterator should really be container - begin and end get the iterators.

Reply:
true, will change.